manifests/server/conf.pp

```diff
@@ -4,12 +4,14 @@
 #
 # Regarding: POODLE - CVE-2014-3566
 #
-# The ``tls_cipher_suite`` parameter is set to ``HIGH:-SSLv2`` because OpenLDAP
-# cannot set the SSL provider natively.
+# Using module defaults and openldap-servers >= 2.4.40, a minimum bound of TLS
+# v1.2 will be set.  TLSv1, SSLv3, and SSLv2 ciphers will be removed from the
```

Comment:
TLSv1 -> TLSv1.0 ?  However, TSLv1.0 is not a cipher list known to openssl on our test infrastructure.  See comment below about TLSv1.0, SSLv2.